modules/aws/agentless-gw/variables.tf

```diff
@@ -124,6 +140,12 @@ variable "binaries_location" {
   nullable    = false
 }
 
+variable "tarball_url" {
```

Comment:
Please change it to binaries_url in order to be consistent with binaries_location

Reply:
Please ignore my last comment and keep it tarball_url